### PR TITLE
Create getting-started-azure.md

### PR DESCRIPTION
**Pull request by tinuxnet**
Missing page as provided for other platforms.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/789)

### CODE REVIEW

**Comment by tinuxnet** [comment]
I signed it!

File: content/docs/started/getting-started-azure.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
++++ title = "AWS For Kubeflow"
```
Comment (sarahmaddox) [inline content/docs/started/getting-started-azure.md]: s/AWS/Azure

Comment (sarahmaddox) [inline content/docs/started/getting-started-azure.md]: While you're here, please would you add the relevant bullet point to this page too:
https://www.kubeflow.org/docs/started/getting-started/

Comment (tinuxnet) [reply content/docs/started/getting-started-azure.md]: New pull requested :)

Comment (tinuxnet) [reply content/docs/started/getting-started-azure.md]: Done

**Comment by sarahmaddox** [comment]
Thanks!

/lgtm
/approve

**Pull request merged**